skimage/io/collection.py

```diff
@@ -104,6 +98,9 @@ class ImageCollection(object):
 
     Notes
     -----
+    Note that files are always stored in alphabetical order. Also note that
```

Comment:
This is false, it's *alphanumeric* order, `im1.jpg` will come before `im10.jpg`. My second contribution ever to scikit-image. =D